.github/workflows/ondemand-pr-build.yaml

```diff
@@ -0,0 +1,64 @@
+name: on-demand build the copr way
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  build_el8:
+    name: when the label is added, build the rpm repo for the PR
+    if: ${{ github.event.label.name == 'build artifacts' }}
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream8
```

Comment:
Still not ready for buildcontainer? What's missing? It would remove a lot of stuff from herr...

Reply:
`jq` is missing -- https://github.com/oVirt/buildcontainer/pull/4

I can convert it over.

Reply:
Using the build container now!